Include/internal/pycore_pystate.h

```diff
@@ -190,10 +190,17 @@ static inline void
 _Py_EnsureFuncTstateNotNULL(const char *func, PyThreadState *tstate)
 {
     if (tstate == NULL) {
+#ifndef Py_GIL_DISABLED
         _Py_FatalErrorFunc(func,
             "the function must be called with the GIL held, "
             "after Python initialization and before Python finalization, "
             "but the GIL is released (the current Python thread state is NULL)");
+#else
+        _Py_FatalErrorFunc(func,
+            "the function must be called with an active thread state, "
+            "after Python initialization and before Python finalization, but it is NULL "
```

Comment:
The "it" in "but it is NULL" seems ambiguous to me. Either:

* Just remove the phrase "but it is NULL"
* Change it to something like "but the thread state is NULL" or "but there is no active thread state"

Reply:
+1 for "but there is no active thread state"